src/rpdk/core/contract/suite/handler_commons.py

```diff
@@ -1,4 +1,5 @@
 import logging
+import copy
```

Comment:
think `isort` is failing on this
```suggestion
import copy
import logging
```

Reply:
Good catcj. Fixed.